data/yam/autoyast/support_images/sles12sp5_install_default_patterns_ppc64le.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+    <suse_register>
+        <do_registration config:type="boolean">true</do_registration>
+        <reg_code>{{SCC_REGCODE}}</reg_code>
+        <install_updates config:type="boolean">true</install_updates>
+    </suse_register>
+    <bootloader>
+      <global>
+          <timeout config:type="integer">-1</timeout>
+      </global>
+    </bootloader>
+    <general>
+      <mode>
+        <confirm config:type="boolean">false</confirm>
+      </mode>
+      <signature-handling>
+        <accept_unsigned_file config:type="boolean">true</accept_unsigned_file>
+        <accept_unknown_gpg_key config:type="boolean">true</accept_unknown_gpg_key>
+        <accept_non_trusted_gpg_key config:type="boolean">true</accept_non_trusted_gpg_key>
+        <import_gpg_key config:type="boolean">true</import_gpg_key>
+      </signature-handling>
+    </general>
+    <networking>
+      <interfaces config:type="list">
+        <interface>
+          <bootproto>dhcp</bootproto>
+          <device>eth0</device>
+          <dhclient_set_default_route>yes</dhclient_set_default_route>
+          <startmode>auto</startmode>
+        </interface>
+      </interfaces>
+    </networking>
+    <firewall>
+      <enable_firewall config:type="boolean">true</enable_firewall>
```

Comment:
Is it required for `qemu` backend？

Reply:
IMO, we need to open the ssh port for s390x platform only

Reply:
keep it after we discuss this on slack. thanks